cernopendata/modules/theme/assets/semantic-ui/js/search/components/TermListItem.js

```diff
@@ -37,7 +38,9 @@ const CODTermListItem = ({ result: { metadata }, index }) => {
       <Item.Content>
         <Item.Header href={link}>{metadata.anchor}</Item.Header>
         <Item.Description href={link}>
-          <p>{truncate(metadata.definition, { length: 200 })}</p>
+          <p>
```

Comment:
This solves the described problem for the LHCb terms, but it breaks the display of general glossary terms such as "Muon", that now have no description.

Basically, we need to check whether the term has the "short_description" field defined, and if yes, then we use it (and strip HTML); otherwise we have to use the regular "description" field (where there may not be the necessity to strip HTML).

Please see the display logic for glossary definition fields in the `qa` branch in `cernopendata/static/templates/search/briefs/terms.html`.

Reply:
Nice catch! I solved it by checking if `short_description` exists. If it doesn't, it uses the standard definition. Stripping the HTML is not necessary anymore in any case.